.ruby-version

```diff
@@ -1 +1 @@
-2.7.6
+3.0.0
```

Comment:
I was able to install 3.0.4 on DC3001
First I ran 
`rvm get stable`
Then verified rvm can get 3.0.4 by running 
`rvm list remote all` as `rvm list known` only returns some ruby versions but not all
then I ran 
`rvm install 3.0.4 -C --with-jemalloc`